public/static/docs/command-reference/checkout.md

```diff
@@ -94,6 +83,13 @@ be pulled from remote storage using `dvc pull`.
   remove files that don't match those DVC-file references or are missing from
   cache. (They are not "committed", in DVC terms.)
 
+- `--relink` - ensures the file linking strategy (`reflink`, `hardlink`,
+  `symlink`, or `copy`) for all data in the workspace is consistent with the
+  project's [`cache.type`](/doc/command-reference/config#cache). This is
+  achieved by restoring **all data files or a directories** referenced in
+  current DVC-files (regardless of whether they match a current DVC-file). Note
+  that this overwrites the data in the workspace.
```

Comment:
overwrites probably not a good term when we deal with links .. it sounds too scary

Reply:
I did mean it as a "scary" warning, but I guess it's a pretty safe operation so perhaps this whole sentence is not needed. Removed!